probe-rs/src/debug/mod.rs

```diff
@@ -339,7 +365,11 @@ impl DebugInfo {
 
         use gimli::Section;
 
-        let frame_section = gimli::DebugFrame::load(load_section).unwrap();
+        let mut frame_section = gimli::DebugFrame::load(load_section).unwrap();
+
+        // To support DWARF v2, where the address size is not encoded in the .debug_frame section,
```

Comment:
Are we sure it's good to hardcode this? Then again, there is no other way of knowing I guess. But maybe we should issue a warning?

Reply:
This is only an issue if we start supporting targets with different address size than 32 bit. Right now we don't, so I don't see this as a problem.

Reply:
I am excited for the moment we are looking for this bug ;) That's why I would add a warning on the commandline :) doesn't bother anyone.

Reply:
Yes, technically RISCV can use 64 bit. I'm not sure any embedded processors do use it, however. Everything in the debug part is hard coded to ARM anyway right now, so currently this cannot happen.

Reply:
Ok. Fair enough. I would still prefer a log statement here, unless it spams the entire console :)